transcom/mymove
# B-18329 Remove N/A customer's address

## B-18329

## Summary
Integration PR: https://github.com/transcom/mymove/pull/12196

## Verification Steps for Reviewers

These are to be checked by a reviewer.


### How to test

1. Create PPM move.
2. Edit and submit move as SC. Take note of pickup and destination address.
3. As TOO find move and click "Move task order" tab to confirm addresses

## Screenshots
<img width="1034" alt="Screenshot 2024-03-13 at 8 33 00 AM" src="https://github.com/transcom/mymove/assets/146969726/3558c462-999f-498a-b3ea-f9fa4b8e73ca">
<img width="1013" alt="Screenshot 2024-03-13 at 8 29 16 AM" src="https://github.com/transcom/mymove/assets/146969726/bb8194a6-89ad-4f20-947e-e22972d37196">
